tests/unit/dbapi_test.py

```diff
@@ -0,0 +1,133 @@
+import unittest
+
+from hazelcast.config import Config
+from hazelcast.db import _make_config, InterfaceError
+
+
+class DbApiTestCase(unittest.TestCase):
```

Comment:
```suggestion
class DbApiTest(unittest.TestCase):
```

to be consistent with our other tests

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/586/commits/6f0929e54cd6bac8cc755b1bae86fc73245df4a1